app/views/reports/_form.html.erb

```diff
@@ -10,6 +10,24 @@
     <%= form.label :category, class: "form-label" %>
     <%= form.select :category, options_for_select(report_options), { include_blank: "Select one..." }, class: "form-select", data: { action: "reveal-by-select" }, required: true %>
 
+    <div data-reveal-by-select-target="There's a bug in this code" class="visibility-hidden">
+      <div class="block mt-1/4 text-small text-dark">
+        <strong>Do not use this form!</strong>
+        <u>Instead, please contact the original creator directly!</u>
+      </div>
+    </div>
+
+    <div data-reveal-by-select-target="I can't import this code" class="visibility-hidden">
+      <div class="block mt-1/4 text-small text-dark">
+        Do not report if:
+
+        <ul class="mt-1/8 pl-1/2">
+          <li>This code is less than 6 months old and has not expired.</li>
+          <li>This code not for PTR.</li>
```

Comment:
This sentence is a little awkward

Reply:
I'd suggest "This code is made for PTR" for the second sentence

Reply:
Would this work better?
Do not report if:
- This code has expired
- This code is made for PTR

Reply:
Yes, perfect :)